### DISCUSSION TITLE
Disable Stripes from uninterruptible cast

### DISCUSSION THREAD

**Discussion by SynaX1** [Backupiseasy/ThreatPlates#305] [Ideas]
I'm new to ThreatPlates and don't find the option to disable this stripes on the castbar border,
when the spell ist not interruptible.

Edit: okay find the Option, can be deleted

**Comment by Backupiseasy** [comment]
The option to disable the stripe texture for the castbar can be found here: 

- Open the options with /tptp
- Go to General - Castbar, Tab Appearance
- Disable "Interrupt Overlay" under Format there.

**Comment by Backupiseasy** [comment]
Syna, sorry that I did not reply here. Obviously, there is no notification for new discussions and it also does not show in the menu.
